db/migrate/20171206121740_create_receivers.rb

```diff
@@ -0,0 +1,14 @@
+class CreateReceivers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :receivers do |t|
+      t.string :name
+      t.text :letter
+      t.references :instituition, foreign_key: true
+      t.integer :number_gifts
```

Comment:
I'm not saying this is wrong but can you explain me what each of these `*_gifts` variable is supposed to be?

Reply:
Haven't made changes on this yet what do you think ?

Reply:
Yeah, when I mentioned "nr de prendas atribuídas", I was referring to `matched_gifts` 😛

Reply:
Yeha, you can calculate the `matched_gifts` but its easier if we pre-calculate it, otherwise we'll end up always traversing a table to count how many times a receiver has been matched. Having that parameter explicitly also allows us to sort by it and give priority to those that have less matches, when selecting a random receiver to match. What do you think?